taxonomy-manager-rest-server/src/main/resources/application.properties

```diff
@@ -10,6 +10,11 @@ quarkus.flyway.connect-retries=10
 quarkus.flyway.table=flyway_quarkus_history
 quarkus.flyway.locations=db
 
+quarkus.smallrye-jwt.enabled=true
+quarkus.smallrye-jwt.auth-mechanism=MP-JWT
+mp.jwt.verify.publickey.location=META-INF/publicTestKey.pem
+mp.jwt.verify.issuer=https://dev.taxman.digirati.com/
```

Comment:
We need to come up with a better way of managing this - the environment (i.e. `dev`) can be passed at runtime, perhaps as an environmental variable. But hardcoding it here is no use, this will break as soon as we deploy to any environment other than `dev`.

In fact, we shouldn't be hardcoding this value at all, including the hostname.

Reply:
I hardcoded this with the intention of having the deploy overwrite this, but it looks like quarkus supports having environment variables in here natively so I think that would better